src/components/new-request/search.vue

```diff
@@ -291,6 +311,12 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
       this.isSupportedAmalgamation(this.getEntityTypeCd)
     ) return true
 
+    if (
+      this.isConversion &&
+      this.isNumberedCompany &&
+      this.isSupportedAlteration(this.getConversionType)
+    ) return true
+
```

Comment:
@severinbeauvais Was this sort of how you pictured us implementing logic for each flow? A separate if statement as needed?

Reply:
Yup, exactly 👍